tests/test_button.py

```diff
@@ -46,6 +47,22 @@ def test_simple_button_registration_fails_when_arguments(mock_route):
     mock_route.assert_not_called()
 
 
+@patch("bottle.Bottle.route")
+def test_group_simple_button_registration(mock_route):
```

Comment:
Not sure if I need a separate test, or if I simply need to test the addition of the parameter. I also did it for both buttons, but I feel like it has code smell. Seems like a lot of duplication.